cmd/run.go

```diff
@@ -432,21 +432,21 @@ func init() {
 
 // Creates a new runner.
 func newRunner(
-	src *loader.SourceData, typ string, filesystems map[string]afero.Fs, rtOpts lib.RuntimeOptions,
+	logger *logrus.Logger, src *loader.SourceData, typ string, filesystems map[string]afero.Fs, rtOpts lib.RuntimeOptions,
```

Comment:
Because chaging `lib.State.Logger` to `logrus.FieldLogger` turned out to be a way too big of a change, that I didn't want to waste a day on. And this where it comes from